src/main/java/DeadlineCommand.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Class to run the deadline command.
+ */
+public class DeadlineCommand implements Command {
+
+    protected final String COMMAND;
+
+    /**
+     * constructor
```

Comment:
I think `constructor` should be capitalised (`Constructor`). I have noticed this in other classes' constructors as well.